.gitignore

```diff
@@ -1,3 +1,6 @@
 /coverage.html
 /coverage.txt
-/TODO.txt
\ No newline at end of file
+/TODO.txt
+.idea
+.vscode
+.DS_Store
```

Comment:
I hope it's okay that I added a few entries to the `.gitignore` file

Reply:
Actually it should be added on your global `.gitignore`, but I guess I'm one the few doing it lol for the convenience it's ok 👍 

Could you remove the `TODO.txt` entry though?

Reply:
I didn't add the `TODO.txt` but I removed it.

And yeah, I know 😅 I've added those 3 lines to almost every project I ever made. Many newcomers don't edit their global `.gitignore` file unfortunately... So I just add this. And in order for me not to forget to add these 3 lines I leave them out of my global `.gitignore` file as well haha